Content.Shared/Storage/Components/MagnetPickupComponent.cs

```diff
@@ -1,23 +1,34 @@
 using Content.Shared.Inventory;
+using Robust.Shared.GameStates;
 
-namespace Content.Server.Storage.Components;
+// namespace Content.Server.Storage.Components;
+namespace Content.Shared.Storage.Components;    // Frontier
```

Comment:
it's also probably not necessary to tag everything with frontier, as long as its mentioned in the pr description (also probably include a link to whichever pr added these to this pr's description, if possible)